documentation/man/features/kw-signature.rst

```diff
@@ -0,0 +1,172 @@
+===================================================
+kw-signature - Write commit and patch trailer lines
+===================================================
+
+.. _signature-doc:
+
+SYNOPSIS
+========
+| *kw* *signature* (-s(<string>) | \--add-signed-off-by=(<string>)) [\--verbose] [<sha> | <patch>]
+| *kw* *signature* (-r(<string>) | \--add-reviewed-by=(<string>)) [\--verbose] [<sha> | <patch>]
+| *kw* *signature* (-a(<string>) | \--add-acked-by=(<string>)) [\--verbose] [<sha> | <patch>]
+| *kw* *signature* (-t(<string>) | \--add-tested-by=(<string>)) [\--verbose] [<sha> | <patch>]
+| *kw* *signature* (-C(<string>) | \--add-co-developed-by=(<string>)) [\--verbose] [<sha> | <patch>]
+| *kw* *signature* (-R(<string>) | \--add-reported-by=(<string>)) [\--verbose] [<sha> | <patch>]
+| *kw* *signature* (-f(<sha>) | \--add-fixes=(<sha>)) [\--verbose] [<sha> | <patch>]
+
+DESCRIPTION
+===========
+This feature adds Linux kernel tags to either commits or patches and helps
+to make this kind of task quicker to complete. It's a wrapper to some useful
+usage of the ``git commit --amend``, ``git rebase`` and ``git interpret-trailers``
+commands. By default, it uses the commit currently pointed by `HEAD`
+as the target of the operation if user does not specify one. At least one **--add**
+option is required.
+
+Most kernel tags contain a signature like:
+
+.. code-block:: text
+
+  TAG: NAME <EMAIL>
+
+By default, options that add these types of tags will use your current `user.name`
+and `user.email` from git configuration if nothing is specified in the command line.
+
+Using multiple options combined is possible. For instance, the following
+command will add **Tested-by** and **Signed-off-by**, in this order,
+to the commit's trailers pointed by `HEAD` using `user.name` and
+`user.email` from git config::
+
+  $ kw signature -t -s
+
+The order the trailer lines are added follows the general sequence:
+
+.. code-block:: text
+
+   Reported-by
+   Co-developed-by
+   Acked-by
+   Tested-by
+   Reviewed-by
+   Signed-off-by
+   Fixes
+
+At the same time, new lines are grouped by their signatures. For example:
+
+.. code-block:: text
+
+   $ kw signature -s'Joe Doe <joe@mail.xyz>' -s -t -t'Joe Doe <joe@mail.xyz>'
```

Comment:
Maybe move -s and -t to the beginning, it is weird to see it in the middle. Or this was intentional?